spec/models/program_spec.rb

```diff
@@ -0,0 +1,280 @@
+  end
+
+  describe '.sponsors_skilled_worker_visa?' do
+    it { expect(described_class.sponsors_skilled_worker_visa?).to be_truthy }
+  end
+end
+
+RSpec.describe SCITTProgramme do
+  describe '.funding_type' do
+    it 'returns an ActiveSupport::StringInquirer' do
+      expect(described_class.funding_type).to be_a(ActiveSupport::StringInquirer)
+    end
+
+    it 'returns "fee"' do
+      expect(described_class.funding_type).to eq('fee')
+    end
+
+    it 'responds to fee?' do
+      expect(described_class.funding_type).to be_fee
+    end
+  end
+
+  describe '.sponsors_student_visa?' do
+    it { expect(described_class.sponsors_student_visa?).to be_truthy }
+  end
+
+  describe '.sponsors_skilled_worker_visa?' do
+    it { expect(described_class.sponsors_skilled_worker_visa?).to be_falsey }
+  end
+end
+
+RSpec.describe SCITTSalariedProgramme do
+  describe '.funding_type' do
+    it 'returns an ActiveSupport::StringInquirer' do
+      expect(described_class.funding_type).to be_a(ActiveSupport::StringInquirer)
+    end
+
+    it 'returns "salary"' do
+      expect(described_class.funding_type).to eq('salary')
+    end
+
+    it 'responds to salary?' do
+      expect(described_class.funding_type).to be_salary
+    end
+  end
+
+  describe '.sponsors_student_visa?' do
+    it { expect(described_class.sponsors_student_visa?).to be_falsey }
+  end
+
+  describe '.sponsors_skilled_worker_visa?' do
+    it { expect(described_class.sponsors_skilled_worker_visa?).to be_falsey }
+  end
+end
+
+RSpec.describe PostgraduateTeachingApprenticeship do
+  describe '.funding_type' do
+    it 'returns an ActiveSupport::StringInquirer' do
+      expect(described_class.funding_type).to be_a(ActiveSupport::StringInquirer)
+    end
+
+    it 'returns "apprenticeship"' do
+      expect(described_class.funding_type).to eq('apprenticeship')
+    end
+
+    it 'responds to apprenticeship?' do
+      expect(described_class.funding_type).to be_apprenticeship
+    end
+  end
+
+  describe '.sponsors_student_visa?' do
+    it { expect(described_class.sponsors_student_visa?).to be_falsey }
+  end
+
+  describe '.sponsors_skilled_worker_visa?' do
+    it { expect(described_class.sponsors_skilled_worker_visa?).to be_truthy }
+  end
+end
+
+RSpec.describe TeacherDegreeApprenticeship do
+  describe '.funding_type' do
+    it 'returns an ActiveSupport::StringInquirer' do
+      expect(described_class.funding_type).to be_a(ActiveSupport::StringInquirer)
+    end
+
+    it 'returns "apprenticeship"' do
+      expect(described_class.funding_type).to eq('apprenticeship')
+    end
+
+    it 'responds to apprenticeship?' do
+      expect(described_class.funding_type).to be_apprenticeship
+    end
+  end
+
+  describe '.sponsors_student_visa?' do
+    it { expect(described_class.sponsors_student_visa?).to be_falsey }
+  end
+
+  describe '.sponsors_skilled_worker_visa?' do
+    it { expect(described_class.sponsors_skilled_worker_visa?).to be_falsey }
+  end
+end
```

Comment:
Should we split into multiple files, or is there a preference to keep all same classes under the same file and spec?

Reply:
This was a preference over having the text `it 'returns false'` seemed a waste of key presses 🤣